webview/platforms/qt.py

```diff
@@ -611,8 +643,13 @@ def _create():
     if window.uid == 'master':
         global _app
         _app = QApplication.instance() or QApplication([])
-
         _create()
+        
+        if _multiprocessing:
+            p = Process()
+            p.exec_ = _app.exec_
+            return p
+            
```

Comment:
What is the rationale behind a custom process class? Why not use python's multiprocessing as with GTK?

Reply:
?

Reply:
> Why not use python's multiprocessing as with GTK?

in `platforms.gtk.py` file i added it
this is qt
qt runs even without gui loop(app.exec_)
why we run gui loop?
i created a custom(and simple) `Process` class